modules/python/math.poly/source/math_poly.i

```diff
@@ -129,6 +147,35 @@ typedef math::linear::Vector<double> VectorDouble;
     {
         return math::poly::TwoD<double>(*$self);
     }
+
+    PyObject* __call__(PyObject* x_input, PyObject* y_input)
+    {
+        if (!PySequence_Check(x_input))
+        {
+            PyErr_SetString(PyExc_TypeError,"Expecting a sequence");
+            return NULL;
+        }
+        if (!PySequence_Check(y_input))
+        {
+            PyErr_SetString(PyExc_TypeError,"Expecting a sequence");
+            return NULL;
+        }
+        int N = PyObject_Length(x_input);
```

Comment:
To be pedantic, this should be a `Py_ssize_t` right?  This will make a difference on Windows if you ever had something so long it wouldn't fit in 4 bytes.  This should ripple through the for loop on line 170 and other similar spots below.